participant/app/components/FileUploader.tsx

```diff
@@ -200,7 +200,7 @@ export const FileUploaderForwardRef: React.ForwardRefRenderFunction<
     <FormGroup>
       {children}
       {previews}
-      <Label htmlFor={id}>{currentDocumentString}</Label>
+      <div className="margin-top-3">{currentDocumentString}</div>
```

Comment:
This is the spacing of the USA Label element; should keep visual alignment the same.